### ISSUE TITLE
Data processing for fine-tuning

### ISSUE THREAD

**Issue by yy524** [k2-fsa/OmniVoice#160]
### Checks

- [x] This template is only for research question, not usage problems, feature requests or bug reports.
- [x] I have thoroughly reviewed the project documentation and read the related paper(s).
- [x] I have searched for existing issues, including closed ones, no similar questions.
- [x] I am using English to submit this issue to facilitate community communication.

### Question details

hi，
    In the paper，”In practical scenarios, audio prompts are often recorded in non-ideal conditions, potentially compromised by environmental noise or reverberation. To prevent the model from replicating these
unwanted artifacts, we implement a prompt denoising task [27, 28]. During training, we augment
a subset of the data by injecting synthetic noise and reverberation into the prompt segments. These
samples are paired with a specific instruction token, <|denoise|>. This mechanism compels the
model to disentangle the speaker’s intrinsic voice identity from the acoustic environment, enabling
the synthesis of clean, high-fidelity speech even when conditioned on degraded prompts.“
   If I want to fine-tune, should the data be processed in the same way? I haven’t done any data augmentation so far, and I’ve noticed that the quality of the synthesized audio degrades. Are there any resources or introductions on augmentation?

   Thank you very much~

**Comment by zhu-han** [comment]
It is not necessary to perform data augmentation during fine-tuning. If you observe degraded audio quality, it is most likely caused by low-quality fine-tuning data or improper fine-tuning hyperparameters (e.g., learning rate, training steps). However, if you intend to explicitly enhance the model’s prompt denoising capability, refer to our documentation at https://github.com/k2-fsa/OmniVoice/blob/master/docs/data_preparation_advanced.md to prepare datasets with augmentation applied to the prompt region.

**Comment by yy524** [comment]
Thank you very much~
